### ISSUE TITLE
Pressure Plates and Stairs are completely broken in latest versions, Fabric 1.20.1

### ISSUE THREAD

**Issue by Rebel459** [Goldorion/MCreatorFabricGenerator#476]
### Issue description

See title. Every time I tried to have 2 of the same stair block, it failed to build. Removing the new one would fix the issue.

### How to reproduce this issue?

Make 2 transparent stair blocks. Tested in a large workspace so there could be more factors at play, but after some testing I narrowed it down within my workspace to this specific issue.

### Operating system

Windows

### Details

MCreator 2023.3 + Fabric Generator Plugin 1.20.1 v1.9.9

### Logs

Probably irrelevant but here

```
[C:\Users\REDACTEDNAME\MCreatorWorkspaces\end_reborn\src\main\java\net\mcreator\endgamereborn\init\EndRebornModBlocks.java:66](file:///C:/Users/REDACTEDNAME/MCreatorWorkspaces/end_reborn/src/main/java/net/mcreator/endgamereborn/init/EndRebornModBlocks.java:66): error: cannot find symbol 
      CHORUS_STAIRS = Registry.register(BuiltInRegistries.BLOCK, new ResourceLocation(EndRebornMod.MODID, "chorus_stairs"), new ChorusStairsBlock()); 
                                                                                                                                ^ 
  symbol:   class ChorusStairsBlock 
  location: class EndRebornModBlocks 
[C:\Users\REDACTEDNAME\MCreatorWorkspaces\end_reborn\src\main\java\net\mcreator\endgamereborn\init\EndRebornModBlocks.java:86](file:///C:/Users/REDACTEDNAME/MCreatorWorkspaces/end_reborn/src/main/java/net/mcreator/endgamereborn/init/EndRebornModBlocks.java:86): error: cannot find symbol 
      ChorusStairsBlock.clientInit();
```

### Issue tracker rule checks (please read carefully)

- [X] I have checked that my problem <a href='https://github.com/MCreator/MCreator/issues?q=is%3Aissue' target='_blank'>is not already reported</a>
- [X] I have checked that my problem is not covered on <a href='https://mcreator.net/support/knowledgebase' target='_blank'>Knowledge Base</a> or on <a href='https://mcreator.net/wiki' target='_blank'>MCreator's Wiki</a>
- [X] I have checked that my written content does not violate the <a href='https://mcreator.net/wiki/general-publishing-guidelines' target='_blank'>publishing guidelines</a>

**Comment by Rebel459** [comment]
Also, it isn't the code being too large, I have less than 125 mod elements, a few of which are advancements

**Comment by Rebel459** [comment]
Further weirdness: after multiple attempts by duping different blocks and redoing them into stairs, I got it to build fine. No clue what the cause is, but something's definitely amiss. Sorry for the lack of info here

**Comment by Rebel459** [comment]
The plot thickens: In-game, they still show up as slabs. The problem definitely has something to do with issues with stairs, as the one time it builds they appear as slabs in-game, after being duped from a slab and changed to a stair. So ig it's a separate issue with duping where the code isn't changed to have them as stairs even though mcreator thinks they are, hence why it would build, as internally there were no stairs. Hence the issue is still related to both stairs and having 2 or more of them

**Comment by Rebel459** [comment]
TLDR after a multitude of tests, there is something very wrong and broken with stairs in mcreator as of right now. Can confirm that when you make a new stair type mcreator breaks, and when duping from another block type the code doesn't change in-game to make it a stair, it shows in mcreator as whatever it was before being duped into a stair from another block, or something along those lines

**Comment by Rebel459** [comment]
FINAL EXPLANATION
I've done tons of testing now, and know the exact issues. Duping a block and changing it to a stair does not actually change the block type, and when you try and create a new stair type (such as a brand new block), it fails to build. Meaning that the stair block type does not work at all as it fails to build each time you attempt to make one, and it only appeared that it worked due to the dupe bug that doesn't actually change it to a stair.

**Comment by Rebel459** [comment]
These exact dupe and project build issues also occur with Pressure Plates

**Comment by Goldorion** [comment]
Already fixed in master. it's coming with the next patch.
